pingora-core/src/listeners/mod.rs

```diff
@@ -115,7 +115,7 @@ impl Listeners {
         listeners
     }
 
-    /// Create a new [`Listeners`] with with a TLS (TCP) endpoint with the given address string,
+    /// Create a new [`Listeners`] with a TLS (TCP) endpoint with the given address string,
```

Comment:
sometimes doubled words are intentional -- each of the doubled words identified in this PR appeared to be accidental